app/models/admin.rb

```diff
@@ -0,0 +1,29 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable,
+         :validatable, authentication_keys: [:login]
+
```

Comment:
Kindly make sure you add :confirmable so that the app will ask for confirmation of email after sign up. I can see that its commented out .